support-frontend/test/services/pricing/PriceSummaryServiceSpec.scala

```diff
@@ -33,10 +33,10 @@ class PriceSummaryServiceSpec extends AsyncFlatSpec with Matchers {
       )
 
     val paper = service.getPrices(Paper, List(discountPromoCode))
```

Comment:
This file is just full of price updates in tests because the catalog fixtures have been updated for the first time in ages, ignore it